starter-code/app.js

```diff
@@ -2,4 +2,42 @@ const express = require('express');
 const app = express();
 const Chuck  = require('chucknorris-io');
 const client = new Chuck();
+const expressLayouts = require('express-ejs-layouts');
 
+
+app.set('views', __dirname + '/views');
+app.set('view engine', 'ejs');
+app.set('layout', 'views/index')
+
+app.use(express.static('static'));
+
+app.get('/', (request, response, next) => {
+  response.send('chuck norris');
+});
+
+app.get('/random', (request, response, next) => {
+  client.getRandomJoke().then((data) => {
+    console.log(data);
+    response.render('random', data);
```

Comment:
data is an object and it has more things that the one you want to use. You can do something like:
```
const jokeObject = {
  joke: chuckRes.value
}
res.render('index', jokeObject);
```
Then you will acces from the view with:
```
<%= joke %>
```
Of course, your solution also works, but be careful. If the response is huge you are wasting resources sending everything when you only need a part.